assignment2/models.py

```diff
@@ -80,9 +80,9 @@ def __init__(self, emb_size, hidden_size, seq_len, batch_size, vocab_size, num_l
         self.embedding = nn.Embedding(vocab_size, emb_size)
 
         self.FirstHidden_input = nn.Linear(emb_size, hidden_size)
-        self.FirstHidden_hidden = nn.Linear(hidden_size, hidden_size)
+        self.FirstHidden_hidden = nn.Linear(hidden_size, hidden_size, bias=False)
```

Comment:
It will be good to add a small comment here to explain why we are setting `bias=False`